### DISCUSSION TITLE
Thoughts on Opera GX - Batch img2img doesn't save files (this is not a bug report)

### DISCUSSION THREAD

**Discussion by danaildichev** [vladmandic/sdnext#1464] [Q&A]
I decided to try Opera GX. My experience is rather mehh. I've come to find that img2img batch runs but doesn't save the files it generates. I don't have that problem in Chrome.

txt2img and regular img2img does work for me in Opera GX. Did I miss something like permissions wise in GX's settings I wonder.

I don't really care though I'm planning on ditching GX. Was just curious what other people here would say.

**Comment by armannita** [comment]
I am so sorry for having 0 Comments but long story short you can call it "Spyware" since it connects to Opera.com and by connecting it, opera has your IP Address.

I am asking yourself did you discover Opera GX through a Sponsor or just an Ad?

The Opera Browser has been in **Fake Update Software** for the Flash Player or Java, when you install it, Opera or GX will install and will be your default browser.

So is kinda a Spyware

-Arman

**Comment by vladmandic** [comment]
> img2img batch runs but doesn't save the files it generates

if you have auto-save enabled, save operation is a server-side thing, should not be related to browser at all.
